mlir/test/Dialect/Rock/lowering_blockwise_gemm_v2.mlir

```diff
@@ -2,7 +2,7 @@
 
 func.func @rock_blockwise_gemm_v2_two_results(%matrix : memref<1024xf32, 3>,
                                                 %bufferA : memref<2xvector<2xf32>, 5>, %bufferB : memref<2xvector<2xf32>, 5>,
-                                                %matrixC : memref<2xvector<32xf32>, 5>) {
+                                                %matrixC : memref<4xvector<16xf32>, 5>) {
```

Comment:
Why change this?

Reply:
Because we are using 64x64 mPerWave and nPerWave. This means mRepeats=2 and nRpeats=2, which means that this will result in a  `2*2*vector<16xf32>` output. The `2*2` concerns the repeats and the `vector<16xf32>` concerns the fact that we are doing a reduction and not a broadcast

Reply:
Fair enough, thanks for the explanation. I think I confused your on-going work with mine. I just took a look at the gridwisegemm logic and confirmed result memref type is `mRepeats * nRepeats * imms.size()`.